cpp/dolfinx/fem/Function.h

```diff
@@ -153,39 +153,21 @@ class Function
   /// @brief Interpolate a provided Function.
   /// @param[in] v The function to be interpolated
   /// @param[in] cells The cells to interpolate on
-  /// @param[in] cell_map A map from cells in the mesh associated with `this`
-  /// function to cells in mesh associated with `v`
-  /// @param[in] nmm_interpolation_data Auxiliary data to interpolate on
-  /// nonmatching meshes. This data can be generated with
-  /// generate_nonmatching_meshes_interpolation_data (optional).
-  void interpolate(
-      const Function<value_type, geometry_type>& v,
-      std::span<const std::int32_t> cells,
-      std::span<const std::int32_t> cell_map,
-      const std::tuple<std::span<const std::int32_t>,
-                       std::span<const std::int32_t>,
-                       std::span<const geometry_type>,
-                       std::span<const std::int32_t>>& nmm_interpolation_data
-      = {})
+  /// @param[in] cell_map A map from cells in the mesh associated with \p this
+  /// function to cells in mesh associated with \p v
```

Comment:
'maps' are much clearer if an example is added, e.g.:

"For cell `i` in the mesh associated with `this`, `cell_map[i]` is the index of the same cell but in the mesh associated with `v`."

(I think this is what it means)